.github/workflows/docker.yml

```diff
@@ -0,0 +1,62 @@
+# GitHub Actions workflow to publish Docker images.
+name: "Docker"
+on:
+  push:
```

Comment:
Are we able to pin this to running on commits pushed to only certain branches, or is there a reason for running on push to any branch?

As we usually push branches to create pull requests, this will trigger image pushes for potentially work-in-progress changes.

Reply:
@joshuasing we can do it on specific branches, how about just `main`?